examples/mini-compile/src/Main.hs

```diff
@@ -18,7 +20,7 @@ module Main (main) where
 
 import "ghc-lib" GHC
 import "ghc-lib" Paths_ghc_lib
-#if defined (GHC_MASTER) || defined (GHC_901)
+#if defined (GHC_MASTER) ||  defined (GHC_921) || defined (GHC_901)
```

Comment:
we might want to think about a `MIN_version` like pragma for this. This seems to be getting a bit out of hand.